src/core/i18n/en/translation.en.json

```diff
@@ -1614,6 +1614,10 @@
       "tooltip": {
         "markdown-help": "You can add formatting to your comment,\nfor example:\nBold text: **bold**\nItalic text: *italic*\nHeader: ##Title\nLink: [text](url)",
         "mentions": "Type a few characters to find relevant users"
+      },
+      "vc-interactions": {
```

Comment:
Ignore this minor one, just FYI, we're using camelCase for the new props in the translations.